examples/amazon_bedrock.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+from llm_guard import scan_output, scan_prompt
+from llm_guard.input_scanners import Anonymize, PromptInjection, TokenLimit, Toxicity
+from llm_guard.output_scanners import Deanonymize, NoRefusal, Relevance, Sensitive
+from llm_guard.vault import Vault
+
+# Specify the AWS region, Bedrock agent ID, alias ID, and session ID
+region = ""
+agent_id = ""
+agent_alias_id = ""
+session_id = ""
```

Comment:
Since these are [constants](https://peps.python.org/pep-0008/#constants) they should be capitalized.